var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -215,12 +215,14 @@ class Cmake(Package):
         depends_on("expat")
         depends_on("zlib")
         # expat/zlib are used in CMake/CTest, so why not require them in libarchive.
-        depends_on("libarchive@3.1.0: xar=expat compression=zlib")
-        depends_on("libarchive@3.3.3:", when="@3.15.0:")
-        depends_on("libuv@1.0.0:1.10", when="@3.7.0:3.10.3")
-        depends_on("libuv@1.10.0:1.10", when="@3.11.0:3.11")
-        depends_on("libuv@1.10.0:", when="@3.12.0:")
-        depends_on("rhash", when="@3.8.0:")
+        for plat in ["darwin", "cray", "linux"]:
```

Comment:
Last time I looked these dependencies were platform-independent?

Reply:
None of them build on Windows due to their current Spack buildsystems, and I'd rather not hold up these PRs waiting for me to port them to CMake. `rhash` in particular has a VS10 build system, which in ancient and would require a non trivial amount of time to port.